src/analysis/retail/shaman/elemental/modules/talents/Icefury.tsx

```diff
@@ -45,18 +45,18 @@ class Icefury extends Analyzer {
     when(this.suggestionThresholds).addSuggestion((suggest, actual) =>
       suggest(
         <>
-          You should fully utilize your <SpellLink id={SPELLS.ICEFURY_TALENT.id} /> casts by casting
-          4 <SpellLink id={SPELLS.FROST_SHOCK.id} />s before the{' '}
-          <SpellLink id={SPELLS.ICEFURY_TALENT.id} /> buff expires. Pay attention to the remaining
+          You should fully utilize your <SpellLink id={TALENTS.ICEFURY_TALENT.id} /> casts by
+          casting 4 <SpellLink id={TALENTS.LAVA_BURST_TALENT.id} />s before the{' '}
```

Comment:
again here, should not be lava burst right?

Reply:
you're right. I changed to FROST_SHOCK